### ISSUE TITLE
Option to treat other metric suffixes the same as _total

### ISSUE THREAD

**Issue by alex-hempel** [DirectXMan12/k8s-prometheus-adapter#48]
Background: We have a Springboot app running on K8s, which exposes metrics to be scraped by Prometheus and visualised in Grafana. We are currently retro-fitting HPA based on custom metrics. (By the way, thank you for the invaluable help your walkthrough has provided).

Our app already exposes counter-type metrics which could be used the same way as (example) http_requests_total, however the suffix is different (_count in our case). I am not a dev as such, and to my understanding, the way these metrics have been exposed, it would be rather difficult for us to change the suffix to _total (or anything other than _count).

Long story short, it would be beneficial if we could treat another suffix the same way _total is being treated. Is there a way to do that without changing the adapter code?

**Comment by tcolgate** [comment]
These are likely to be from a histogram or summary.

**Comment by DirectXMan12** [comment]
I've got a PR that makes things much more customizable (#46), so it should enable your usecase.  Can you see if that helps your issue?

**Comment by alex-hempel** [comment]
Hi @DirectXMan12 

that PR looks interesting, I've just cloned it and built a local image.

Do you have an example of how to set up and apply the config file? I assume the file format would be yaml?

**Comment by DirectXMan12** [comment]
ah, yeah, sorry.  I appear to have forgotten to check in the default config -- it's just generated internally right now for compatibility reasons.  I'll check in an example configuration with some comments explaining how things work.

**Comment by DirectXMan12** [comment]
There should be a sample configuration under `docs` now.

**Comment by alex-hempel** [comment]
Thank you for that.

I will close this for now. I haven't had time or opportunity to test this in-depth. Will get back to you when I do.
